### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by R1kaB3rN** [Open-Wine-Components/ULWGL#204]
See https://github.com/Open-Wine-Components/umu-launcher/issues/202#issuecomment-2395585975 for context.

**Comment by FireLemons** [comment]
Yeah I just tried to install it on regular ubuntu using the readme. Spent a few minutes learning about why pip is warning me that I have an "externally-managed-environment". Then installed hatchling a few times which was pretty hard on linux for some reason.

![image](https://github.com/user-attachments/assets/757508fe-5a9c-4712-b842-d474db55f41e)

only for `make`  to keep telling me it can't find `hatchling.build`

Then I google for instructions that someone else may have made for installation on ubuntu. Fortunately, the releases page appears in the search results. So I go download the one marked ubuntu and it contains 2 `.deb` files so I try to install `umu-launcher_1.1.3-1_all.deb` but apparently I need both and the other package has to be installed first.  
  
Sometimes I don't read documentation very carefully and miss a few steps so maybe I missed an alternate set of instructions telling me about the releases.
![image](https://github.com/user-attachments/assets/b33e2bc6-f1be-4f4b-a261-9319cc79e074)
